runtime/browser/xwalk_browser_context.cc

```diff
@@ -349,6 +315,17 @@ net::URLRequestContextGetter*
 #endif
 }
 
+net::URLRequestContextGetter* XWalkBrowserContext::CreateMediaRequestContext() {
```

Comment:
I think so as only GetRequestContext() was used once in xwalk_devtools_frontend.cc:234 which has been replaced by `content::BrowserContext::GetDefaultStoragePartition(web_contents()->GetBrowserContext())->GetURLRequestContext()`